### PR TITLE
Support class agnostic in mask_cross_entropy loss

### PR DESCRIPTION
**Pull request by Zehaos**

### CODE REVIEW

**Comment by hellock** [comment]
Thanks for the fix!

File: mmdet/core/loss/losses.py [hunk 1]
```diff
@@ -56,6 +56,8 @@ def weighted_sigmoid_focal_loss(pred,
 def mask_cross_entropy(pred, target, label):
     num_rois = pred.size()[0]
     inds = torch.arange(0, num_rois, dtype=torch.long, device=pred.device)
+    if pred.shape[1] == 1:
+        label.zero_()
```
Comment (hellock) [inline mmdet/core/loss/losses.py]: We could avoid the inplace modification of `label` in case that it will be used afterwards.

**Comment by hellock** [comment]
Hi @Zehaos The fix is fine, just needs to be implemented in some other way. Would you submit a new PR?

**Comment by Zehaos** [comment]
@hellock I made a new pull request #177

**Pull request closed without merging**